mcs/class/System.Web/System.Web.Util/HttpEncoder.cs

```diff
@@ -30,12 +30,14 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
+#if !UNITY_FORWARD_TO_SYSTEM
```

Comment:
Just curious but would it be possible to just not include HttpEncoder.cs in the unityjit sources?

Reply:
I tried for almost 2 hours to get the "*.excludes.sources" mechanism working but failed. Or we can copy/paste the entire source list and remove a few files.

Reply:
Using the exclude would be best but this is probably better than having yet another copy of all the sources just to exclude a file.

Reply:
@bholmes got the exclude file working for me ✋